proto/ffi.proto

```diff
@@ -25,6 +25,16 @@ message ExecutionRequest {
   bytes input_data = 3;
 }
 
+message CbpfExecutionRequest {
+  // Program pair sockets
+  int64 socket_parent = 1;
+  int64 socket_child = 2;
+
+  // Optional data to send over the network that can be accessed by the
+  // ebpf program.
+  bytes input_data = 3;
+}
+
 // Results from Executing the ebpf program.
 message ExecutionResult {
```

Comment:
for cbpf, consider adding a field for the data that the child socket receives

Reply:
Done